packages/webservice/app/src/main/resources/application.yaml

```diff
@@ -8,6 +8,13 @@ elide:
   swagger:
     path: /doc
     enabled: true
+  async:
+    enabled: true
+    threadPoolSize: 7
+    maxRunTime: 65
+    cleanupEnabled: true
+    queryCleanupDays: 7
+    defaultAsyncQueryDAO: true
 
```

Comment:
we can include the aggregation datastore enable config.

Reply:
👍